librz/io/p/io_srec.c

```diff
@@ -157,8 +203,8 @@ static bool srecord_parse(RzBuffer *buf, char *str) {
 
 	do {
 		line++;
-		if (sscanf(str, "S%c%02x", &record_type, &byte_count) != 2) {
-			RZ_LOG_ERROR("srec:parse(): invalid data in motorola srecord file at line %d\n", line);
+		if (!parse_srecord_prefix(str, &record_type, &byte_count)) {
+			RZ_LOG_ERROR("srec:parse(): invalid prefix in motorola srecord file at line %d\n", line);
```

Comment:
Maybe `Motorola SRecord` instead of `motorola srecord`?

Reply:
Wikipedia says "Motorola S-record", let's use that.